Bottom: d5fde90696525d58f49d13c981b7e982626832b0
Top:    8f4ad70dc805f53cd8009647c7e216c1300644c6
Author: Ian Kent <raven@themaw.net>
Date:   2013-02-27 13:23:42 +0800

Refresh of autofs-5.0.7-fix-requires-in-spec-file.patch

---

diff --git a/autofs.spec b/autofs.spec
index fd4a120..d49a78f 100644
--- a/autofs.spec
+++ b/autofs.spec
@@ -32,8 +32,8 @@ BuildRequires: systemd-units
 %if %{with_libtirpc}
 BuildRequires: libtirpc-devel
 %endif
-BuildPrereq: autoconf, hesiod-devel, openldap-devel, bison, flex, cyrus-sasl-devel
-Prereq: chkconfig
+BuildRequires: autoconf, hesiod-devel, openldap-devel, bison, flex, cyrus-sasl-devel
+Requires: chkconfig
 Requires: /bin/bash mktemp sed textutils sh-utils grep /bin/ps
 %if %{with_systemd}
 Requires(post): systemd-sysv
